weave/trace_server/trace_server_interface.py

```diff
@@ -212,6 +216,18 @@ class TableSchemaForInsert(BaseModel):
     rows: list[dict[str, Any]]
 
 
+class OtelExportReq(BaseModel):
+    model_config = ConfigDict(arbitrary_types_allowed=True)
+    project_id: str
+    traces: ExportTraceServiceRequest
+    wb_user_id: Optional[str] = Field(None, description=WB_USER_ID_DESCRIPTION)
+
+
+# Spec requires that the response be of type Export<signal>ServiceResponse
+# https://opentelemetry.io/docs/specs/otlp/
+OtelExportRes: TypeAlias = "ExportTraceServiceResponse"
```

Comment:
sorry. i should have been specific. Here, please use:

```
class OtelExportRes(BaseModel):
   response: ExportTraceServiceResponse
```

Then you can correctly handle the contract in the server layer. Your `OtelExportReq` already deviates from a pure otel sink

Reply:
interesting. i wonder why it is ok with the request

Reply:
you could also do a TypedDict

Reply:
Nevermind, I'm getting a bad request error pretty much no matter what I do after swapping to this. I think it's going to take more work than just an ignore flag...

Reply:
that comment implies something is not clear. The service itself needn't us this exact payload.